pay-queue/src/pay_queue/services/cgi_reconciliations.py

```diff
@@ -87,7 +87,7 @@ def _update_feedback(msg: Dict[str, any]):  # pylint:disable=too-many-locals, to
     minio_location: str = msg.get("location")
     file = get_object(minio_location, file_name)
     content = file.data.decode("utf-8-sig")
-    group_batches: List[str] = _group_batches(content)
+    group_batches: Dict[str, List[str]] = _group_batches(content)
```

Comment:
it is Dict[str, List[str]]. here is function:
![image](https://github.com/user-attachments/assets/b9f83e1d-ec40-49d7-9270-15195bd6aa0a)